### PR TITLE
Feat: Added Image utilities

### PR DESCRIPTION
**Pull request by kjais1720**
Added utility class `.img-responsive` to make the image scale according to its container

### CODE REVIEW

File: Docs/Images/image.html [hunk 1]
```diff
@@ -0,0 +1,190 @@
...
+        </aside>
+    </div>
+
+    <header class="tr-header main-header d-flex f-wrap">
+        <button data-target="sidebarMain" class="tr-btn tr-btn-icon hamburger"><i class="fas fa-bars"></i></button>
+        <div class="tr-heading">
+            <a href="../../index.html">
+                <h2 class="header-brand">
+                    <img class="logo" src="../../assets/tarang-favicon-2.png" alt="Tarang Logo">
+                    <span>arang</span> <span>UI</span>
```
Comment (SimranjitKaurMaan) [inline Docs/Images/image.html:78]: Typo I guess.

**Review by SimranjitKaurMaan — APPROVED** [review]

File: Docs/Images/image.html (hunk 1 above)
Comment (kjais1720) [reply Docs/Images/image.html:78]: Thank you for pointing it out,I have updated in the latest issue

File: Docs/Images/image.html [hunk 2]
```diff
@@ -0,0 +1,190 @@
...
+                <i class="fab fa-github"></i>
+            </a>
+        </nav>
+    </header>
+
+    <main class="main-content d-flex">
+        <div class="component-desc flex-col">
+            <h2>Images</h2>
+            <p>
+                Images are integral part of any website, they are really important to attract user's attention or give them some information, so handling images and making them responsible becomes really important.
```
Comment (Sameony) [inline Docs/Images/image.html:99]: is that meant to be "responsible" or is it "responsive"..?

**Review by Sameony — COMMENTED** [review]
You can add and commit the html layout from the dev branch before switching to avoid putting it up for every PR . Great code, it's ready to merge. You can work on the text misspellings now or later.

File: Docs/Images/image.html (hunk 2 above)
Comment (kjais1720) [reply Docs/Images/image.html:99]: Responsive, thanks for pointing it out, I have fixed it in the latest commit

**Pull request merged**